Samples/HelloWorld/SiloTests/HelloWorldSiloTests.cs

```diff
@@ -31,20 +31,10 @@ namespace Tests
     ///    testing framework such as NUnit or xUnit could have been used instead.
     /// ----------------------------------------------------------------- */
     /// </summary>
-    [TestClass]
-    public class HelloWorldSiloTests : TestingSiloHost
+    public class HelloWorldSiloTests : TestingSiloHost, IDisposable
```

Comment:
the new recommended approach is to not inherit from `TestingSiloHost`, especially if you use something other than MSTest as here, that has better control for when tear down happens (not just TestCleanup, which is pretty deterministic, but TestCleanup from MSTest is not) .
Also, instead use the new `TestCluster` (instead of `TestingSiloHost`) which favors programmatic config a better cluster lifetime management.